src/main/java/edu/greenblitz/tobyDetermined/Robot.java

```diff
@@ -65,6 +65,9 @@ private static void initSubsystems() {
 		IntakeExtender.init();
 		IntakeRoller.init();
 		OI.init();
+
+//		LED.getInstance(); //todo might make error
+		LED.init(); //todo might make error
```

Comment:
Similar to what Nitzan said about errors, but with LED.init().
In general, there should not be TODO's in master